test/appium/tests/conftest.py

```diff
@@ -236,7 +236,10 @@ def pytest_runtest_makereport(item, call):
 
 def update_sauce_jobs(test_name, job_ids, passed):
     for job_id in job_ids.keys():
-        sauce.jobs.update_job(job_id, name=test_name, passed=passed)
+        try:
+            sauce.jobs.update_job(job_id, name=test_name, passed=passed)
+        except RemoteDisconnected:
+            pass
```

Comment:
should the program exit here or what is expected in case of internal error?

Reply:
internal error caused by RemoteDisconnected exception in pytest_runtest_makereport, the job will keep running